nav/stepper/stepper_constants.py

```diff
@@ -20,13 +20,38 @@
 FL_step = 24
 FL_dir = 23
 
+<<<<<<< HEAD
+# Direction constants
+FWD = True
+REV = False
+
+# Clockwise and Counter-clockwise constants 
+CW = 1 			                 
+CCW = 0
+
+# whole, 1/2, 1/4, and 1/8 steps per revolution constants.
+# steps per revolution is obtained by (360/1.8) * denominator. 
+# ie: (360/1.8) * 2 = 400 for half steps per revolution
+STEPS_PER_REVOLUTION = 200
+HALF_STEPS_PER_REVOLUTION = 400
+QUARTER_STEPS_PER_REVOLUTION = 800 		
+EIGTH_STEPS_PER_REVOLUTION = 1600
+
+=======
 #Direction constants
 FWD = True
 REV = False
 
 CW = 1 			                 # Clockwise rotation
 CCW = 0 		                 # Counter-clockwise rotation
 STEPS_PER_REVOLUTION = 800 		 # Steps per Revolution (360/1.8) * 4. Multiply by 4 because quarter step.
+>>>>>>> upstream/motion
 DISTANCE = 60*math.pi/25.4
+
+# individual steps per inch based off of whole step, 1/2 step, 1/4 step, and 1/8 step.
 STEPS_PER_INCH = STEPS_PER_REVOLUTION/DISTANCE
```

Comment:
Again, this looks clunky. Also, DISTANCE depends on the wheel size as far as I know. @BenjiLee0002 can expand on this. This needs to be dynamically set based on wheel size.

Reply:
Yes, rahul was supposed to implement this change but if you want to do it Jonathan then you can. It  should also be a dict like Rishav suggested for STEP_PER_REVOLUTION.